onoff.js

```diff
@@ -240,6 +241,18 @@ class Gpio {
       // the bbb.
     }
   }
+
+  static get accessible() {
+    try {
+      const p = path.join(GPIO_ROOT_PATH, 'export');
+      fs.openSync(p, 'w');
```

Comment:
The file descriptor returned by `openSync` is leaked. `closeSync` should be called to close the file descriptor.